examples/appendix_a/resources/input.ron

```diff
@@ -1,14 +1,14 @@
 (
     axes: {
-        "left_paddle": (
+        "left_paddle": Emulated(
```

Comment:
Do we absolutely need to change the tuple to an Emulated?

Reply:
This now becomes an enum instead of a struct, so tuple alone is ambiguous. It is necessary, so axes can be bound to `Controller` case too. I chose name `Emulated` because that's how it was referred to in the docs around this feature. If you have any other ideas, please share.

Reply:
Makes sense